.github/workflows/frontend-workflow.yml

```diff
@@ -1,29 +1,12 @@
-name: Continuous-Integration
-on: [push] # This workflow triggers anytime a commit is pushed
+name: Frontend CI
+on:
+  push:
+    branches-ignore:
+      - 'backend/**' # Don't run on any backend branches
 
 jobs:
-  test-backend:
-    name: 'Backend Testing'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@master
-      - name: docker-build
-      # This option makes the Postgres container exit when the autoscheduler container does
-        run: docker-compose up --abort-on-container-exit
-
-  lint-backend:
-    name: 'Backend Linting'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@master
-      - name: pylint
-        uses: cclauss/GitHub-Action-for-pylint@master
-        with:
-        # Installs Django and pylint-django, then runs pylint on all of the files in autoscheduler/
-          args: pip install -r .github/lint-requirements.txt ; pylint autoscheduler/**/*.py --load-plugins pylint_django
-
   test-frontend:
-    name: 'Frontend Testing'

```

Comment:
If we did it this way, then in the checks (like at the bottom of this PR), it would say `Frontend CI / Frontend Testing (push)`, which I thought was redundant since it always includes the workflow name (`Frontend CI` in this case) when listing the checks. Do you think doing it like that helps the clarity regardless?

Reply:
Oh no, it's fine in that case. For whatever reason, I was blind and didn't see the Frontend CI prefix. Looks good to me!